data/built-ins.json

```diff
@@ -671,28 +712,31 @@
     "edge": 14,
     "firefox": 43,
     "safari": 10,
+    "node": 6,
     "ios": 10,
     "opera": 34
   },
   "es7.object.values": {
     "chrome": 54,
     "edge": 14,
     "firefox": 47,
+    "safari": 10.1,
     "node": 7,
     "opera": 41
   },
   "es7.object.entries": {
     "chrome": 54,
     "edge": 14,
     "firefox": 47,
+    "safari": 10.1,
     "node": 7,
     "opera": 41
   },
   "es7.object.get-own-property-descriptors": {
     "chrome": 54,
     "edge": 15,
     "firefox": 50,
-    "safari": 10,
```

Comment:
Hm if it's a top level thing it's supposed to match on both

like in "es6.array.iterator"